### ISSUE TITLE
super worked in older browsers without loose mode, now doesn't?

### ISSUE THREAD

**Issue by lukescott** [6to5/6to5#741]
Before 726451f86fcb69a32abb3075c361830c3f9644fe super worked as far back as IE 8, using core-js of course. After 726451f86fcb69a32abb3075c361830c3f9644fe my tests started failing. If I enable loose mode it works again, but then my methods are enumerable:true in IE 9 and above, instead of the spec compliant false.

I couldn't find any reference to that commit, other than #690. What was the reason for going from:
```javascript
_get(One.prototype, "constructor", this).call(this);
```
To:
```javascript
_get(Object.getPrototypeOf(Two.prototype), "constructor", this).call(this);
```

For example, this was working in IE8 and now it's not:
```javascript
class One {
  constructor() {
    this.test = "test";
  }
}

class Two extends One {
  constructor() {
    super();
  }
}

console.log((new Two()).test);
```

In [cavets](https://6to5.org/docs/usage/caveats/) it says that super won't resolve correctly in IE 10 or below. Ignoring that it did work in IE 8, it currently works in IE 9 (without loose mode), at least it does in my project. Does that seem correct? Should that page be updated?

**Comment by sebmck** [comment]
I was under the assumption for a short amount of time that the `HomeObject` was set at definition time and it was changed back as it's more inline with the spec.

The reason the caveats says that super wont resolve correctly is that super constructors **will** but the following **wont**:

```javascript
class Foo {
  foo() { return "foo"; }
}

class Foo extends Bar {
  foo() { return super.foo() + "bar"; }
}
```

**Comment by lukescott** [comment]
@sebmck 

Is `class Foo extends Bar` supposed to be `class Bar extends Foo`. If so, that appears to work to me:

https://6to5.org/repl/#?experimental=false&playground=false&evaluate=true&loose=true&code=class%20Foo%20%7B%0A%20%20foo()%20%7B%20return%20%22foo%22%3B%20%7D%0A%7D%0A%0Aclass%20Bar%20extends%20Foo%20%7B%0A%20%20foo()%20%7B%20return%20super.foo()%20%2B%20%22bar%22%3B%20%7D%0A%7D%0A%0Aconsole.log((new%20Bar).foo())%3B

**Comment by sebmck** [comment]
@lukescott Whoops yeah it is. Not sure, I've had reports that it doesn't work and if I recall correctly whenever I've tested it hasn't. I'm not really willing to excessively document browser support, especially in such older browsers, because I'm going to have to maintain and support that.

**Comment by lukescott** [comment]
@sebmck I switched back to strict mode. core-js just released a fix (0.5.2) for Object.getPrototypeOf that returns the correct value. Other than no getter/setter support (not using those intentionally), I'm not sure what doesn't work. My tests pass all the way back to IE 7 now.

I know you don't care about older browsers, but if there are more caveats, it would be nice to have them documented. Just out of curiosity, how do you run tests across browsers? Do you use something like Browserstack?
